### PR TITLE
[PR-16667] Support encryption for origin params

### PR DESCRIPTION
**Pull request by max-si-m**
## Demo
https://deploy-preview-262--talkable-docs.netlify.app/advanced_features/events.html#available-properties
https://deploy-preview-262--talkable-docs.netlify.app/integration/custom/integration_components.html#post-purchase-script-notes

## Related Stories
[](https://talkable.atlassian.net/browse/PR-16667)

### CODE REVIEW

**Review by hmlON — COMMENTED** [review]
I think it looks weird. I would prefer `here` to be the link. WDYT?
image

File: source/advanced_features/events.rst [hunk 1]
```diff
@@ -73,6 +73,10 @@ in the Reports → Purchases section of your Talkable Dashboard. Here is an exam
 Available properties
 ~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+   All PII params supports data encryption, more information can be found here :ref:`Params Encryption <advanced_features/params_encryption>`.
```
Comment (zhuravel) [inline source/advanced_features/events.rst:78]:
```suggestion
   All PII params support data encryption, more information can be found here :ref:`Params Encryption <advanced_features/params_encryption>`.
```

File: source/integration/custom/integration_components.rst [hunk 2]
```diff
@@ -153,6 +153,10 @@ Post Purchase Script Notes
 6. If you need to pass shopping cart line items, to see the alternate
    post purchase integration script :ref:`click here <advanced_features/product_items>`.
 
+.. note::
+
+   All PII params supports data encryption, more information can be found here :ref:`Params Encryption <advanced_features/params_encryption>`.
```
Comment (zhuravel) [inline source/integration/custom/integration_components.rst:158]:
```suggestion
   All PII params support data encryption, more information can be found here :ref:`Params Encryption <advanced_features/params_encryption>`.
```

File: source/advanced_features/events.rst [hunk 3]
```diff
@@ -73,6 +73,10 @@ in the Reports → Purchases section of your Talkable Dashboard. Here is an exam
 Available properties
 ~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+   All PII params supports data encryption, more information can be found on :ref:`Params Encryption <advanced_features/params_encryption>`.
```
Comment (zhuravel) [inline source/advanced_features/events.rst:78]:
```suggestion
   All PII params support data encryption, more information can be found on :ref:`Params Encryption <advanced_features/params_encryption>`.
```

**Review by zhuravel — CHANGES_REQUESTED** [review]

File: source/advanced_features/events.rst [hunk 4]
```diff
@@ -73,6 +73,10 @@ in the Reports → Purchases section of your Talkable Dashboard. Here is an exam
 Available properties
 ~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+  All PII params support data encryption. Find more about :ref:`Params Encryption <advanced_features/params_encryption>`.
```
Comment (zhuravel) [inline source/advanced_features/events.rst:78]:
```suggestion
   All PII params support data encryption. Find more about :ref:`Params Encryption <advanced_features/params_encryption>`.
```

**Review by zhuravel — APPROVED** [review]

**Review by hmlON — APPROVED** [review]

**Pull request merged**